css/impact.css

```diff
@@ -0,0 +1,286 @@
+Anything that needs to change for larger screens should be added to the media queries below.
+
+Feel free to change the min/max-width values to whatever you need. Use variables.
+===================================
+*/
+
+/* Tablet Only */
+@media screen and (min-width: 768px) and (max-width: 1024px) {
+	/* Add styles that only apply to tablet. These are usually fixes to get resposnivenes right */
+}
+
+/* Tablet and Larger */
+@media screen and (min-width: 768px) {
+	/* Add styles that apply to table and desktop */
+	/* Title/Hero section */
+	.title-img {
+		display: none;
+	}
+	/* adding desktop image */
+	.hero {
+		height: 100vh;
+		max-height: 1000px;
+		background-image: url("/assets/images/pages/our-impact/desktop/vr-ar-d.jpg");
+		background-size: cover;
+		background-position: center;
+		background-repeat: no-repeat;
+	}
+	.title-text {
+		width: 100%;
+		height: 100%;
+		background: linear-gradient(
+			92deg,
+			rgba(9, 9, 9, 0.77) 40.35%,
+			rgba(9, 9, 9, 0.74) 44.51%,
+			rgba(9, 9, 9, 0.71) 48.63%,
+			rgba(9, 9, 9, 0) 83.13%
+		);
+
+		/* aligning the title text div */
+		display: flex;
+		align-items: center;
+	}
+	.title-text div {
+		/* adding width instead of margin for responsiveness */
+		width: 70%;
+		margin: 0 auto;
+
+		display: flex;
+		flex-direction: column;
+	}
+	.title-text p,
+	.title-text .h2-styles {
+		/* adding width to make sure text stays on left side */
+		width: 40%;
+		text-align: left;
+		color: var(--color-white);
+	}
+	.title-text .h2-styles {
+		font-size: var(--font56);
+		line-height: var(--line-height58);
+	}
+
+	
+
+	/* icon section */
+	.icons-parent {
+		flex-direction: row;
+		flex-wrap: wrap;
+		justify-content: center;
+	}
+	.icon-child {
+		min-width: 40%;
+	}
+	.yellow-background {
+		background-size: auto 90%;
+	}
+}
+
+/* Desktop Only */
+@media screen and (min-width: 1024px) {
+	/* Add styles that apply specifically to desktop */
+  
+	/* top and bottom section */
+
+	.tech-around,
+	.responsible-parent {
+		width: var(--desktop-wrapper-width);
+		max-width: var(--wrapper-width-max);
+		margin: var(--spacing100) auto;
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+		gap: var(--spacing100);
+	}
+	.tech-around > * {
+		width: 50%;
+	}
+	.responsible-parent > * {
+		width: 50%;
+	}
+
+	.red-background {
+		background-color: transparent;
+		position: relative;
+	}
+
+	.tech-around .red-background,
+	.responsible-parent .red-background {
+		max-width: 500px;
+		margin: 0 auto;
+	}
+	/* tech around section */
+	.red-block-one {
+		background-color: var(--color-amaranth);
+		border-radius: var(--spacing10);
+		display: block;
+		min-width: 200px;
+		min-height: 200px;
+		position: absolute;
+		transform: translate(-45px, -235px);
+	}
+
+	.tech-img {
+		margin: 0px;
+		position: absolute;
+		left: 55%;
+		width: 425px;
+		transform: translate(-50%, -50%);
+	}
+
+	/* responsible section */
```

Comment:
Noticed a few inconsistencies from the design for text!

In desktop, the title text needs to be aligned left:
![image](https://github.com/indigenousfriends/itlct-website-redesign/assets/98284615/2f36d8d6-819d-4114-a80d-4d4256aaae23)


In mobile, i think we need a wrapper or some padding! Take a look at what was done in the tech-around info area:

![image](https://github.com/indigenousfriends/itlct-website-redesign/assets/98284615/bc19293a-7b64-47c2-9902-2ed8038202a4)

Reply:
all addressed!